xhyve/xhyve.go

```diff
@@ -612,42 +612,44 @@ func (d *Driver) publicSSHKeyPath() string {
 	return d.GetSSHKeyPath() + ".pub"
 }
 
-func readLine(path string) string {
-	line := ""
+func readLine(path string) (string, error) {
 	inFile, err := os.Open(path)
 	if err != nil {
-		log.Debugf("Not able to open %s", path)
+		return "", err
 	}
 	defer inFile.Close()
+
 	scanner := bufio.NewScanner(inFile)
-	scanner.Split(bufio.ScanLines)
 	for scanner.Scan() {
-		line = scanner.Text()
-		if kernelOptionRegexp.MatchString(line) {
-			break
+		if kernelOptionRegexp.Match(scanner.Bytes()) {
+			m := kernelOptionRegexp.FindSubmatch(scanner.Bytes())
+			return string(m[1]), nil
```

Comment:
@zchee I did `go install` after putting your changes to my current branch  that's why it's showed (HEAD), I  think if I used your branch or rebase mine against yours then it will show commit hash. I will try later one from next time.